app/uploaders/logo_uploader.rb

```diff
@@ -0,0 +1,49 @@
+class LogoUploader < BaseUploader
+  MAX_FILE_SIZE = 3 # Megabytes
```

Comment:
I think self-documenting code is better than comments, so let's move the call to `megabytes` up here and remove it further down the file.

```suggestion
  MAX_FILE_SIZE = 3.megabytes
```

Reply:
✅ Thanks this makes a lot more sense.

Reply:
I had done this explicitly to make things easier on the front-end, but I'm all good with this change.

Reply:
Sorry @nickytonline I should have considered that, feel free to change it back in a subsequent PR when you add the validation. I think it's really beneficial to keep track of the value in a single place like you suggested.